setup.cfg

```diff
@@ -1,9 +1,19 @@
 [flake8]
-exclude = *migrations*,
-    kitsune/settings*,
-    kitsune/sumo/db_strings.py,
+exclude =
```

Comment:
This break is just ugly :)

Reply:
We'll need to find a formatter for our *.cfg files then ;)